.github/ISSUE_TEMPLATE/recurring_bug.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Recurring Bug
+about: Bug seen before/across multiple projects
+title: 'Recurring Bug'
```

Comment:
I don't think this is necessary. We should either a) re-use the Bug template and reference the old/original bug or b) re-open the old/original bug template. My preference would be to do b) as we can have the history in one place. Github will show the issue re-opened and then the work can resume there.

Reply:
let's discuss in the projects meeting today. in our meeting last week there was pretty strong support for this idea.